lua/cmp/utils/highlight.lua

```diff
@@ -1,46 +1,30 @@
 local highlight = {}
 
 highlight.keys = {
-  'gui',
-  'guifg',
-  'guibg',
-  'cterm',
-  'ctermfg',
-  'ctermbg',
+  'fg',
+  'bg',
+  'bold',
+  'italic',
+  'reverse',
```

Comment:
Isn't it `inverse` for that API?

Reply:
For that API it is actually `reverse`. I found this yesterday when `nightfox.nvim` moved to using the `nvim_set_hl` API: https://github.com/EdenEast/nightfox.nvim/issues/136